### PR TITLE
task 2

### PR DESCRIPTION
**Pull request by webdevelopland**
What do you think?

### CODE REVIEW

**Comment by lidebug** [comment]
Wow hah

File: projects/angular/routes/home/csv/crop/crop.component.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Component, Output, EventEmitter } from "@angular/core";
+import {FormControl} from '@angular/forms';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/operator/startWith';
+import 'rxjs/add/operator/map';
```
Comment (lidebug) [inline projects/angular/routes/home/csv/crop/crop.component.ts]: Why do you user map here?

File: projects/angular/routes/home/csv/crop/crop.component.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  selector: "crop",
+  templateUrl: "./crop.html"
+})
+export class CropComponent {
+  //It's a pretty common example from material.angular.io
+  @Output() load = new EventEmitter();
+  crop: FormControl = new FormControl();
+  msg:string = "";
+
+  options = [
```
Comment (lidebug) [inline projects/angular/routes/home/csv/crop/crop.component.ts]: mb you should load it from server?

File: projects/angular/routes/home/csv/csv.component.ts [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import { Component, ViewChild } from "@angular/core";
+import { FirebaseService } from "@/services/firebase.service";
+import { is, not } from "@/shell/is";
```
Comment (lidebug) [inline projects/angular/routes/home/csv/csv.component.ts]: is and not?

File: projects/angular/routes/home/csv/crop/crop.component.ts (hunk 2 above)
Comment (webdevelopland) [reply projects/angular/routes/home/csv/crop/crop.component.ts]: let's see

File: projects/angular/routes/home/csv/crop/crop.component.ts (hunk 1 above)
Comment (webdevelopland) [reply projects/angular/routes/home/csv/crop/crop.component.ts]: yeah, I'll delete it

File: projects/angular/routes/home/csv/csv.component.ts (hunk 3 above)
Comment (webdevelopland) [reply projects/angular/routes/home/csv/csv.component.ts]: it's good

**Pull request merged**